spec/open_weather/endpoints/stations_spec.rb

```diff
@@ -94,4 +94,68 @@
       end
     end
   end
+
+  describe '#create_measurements' do
+    it 'creates measurements', vcr: { cassette_name: 'stations/create_measurement_success' } do
+      create_params = {
+        "station_id": '5ed21a12cca8ce0001f1aef1',
+        "dt": 1479817340,
+        "temperature": 18.7,
+        "wind_speed": 1.2,
+        "wind_gust": 3.4,
+        "pressure": 1021,
+        "humidity": 87,
+        "rain_1h": 2,
+        "clouds": [
+          {
+            "condition": 'NSC'
+          }
+        ]
+      }
+      data = client.create_measurements([create_params])
+      expect(data).to be_nil
+    end
+
+    context 'when station does not exist' do
+      it 'raises error', vcr: { cassette_name: 'stations/create_measurement_failed_with_invalid_station' } do
+        create_params = {
+          "station_id": -1,
+          "dt": 1479817340,
+          "temperature": 18.7,
+          "wind_speed": 1.2,
+          "wind_gust": 3.4,
+          "pressure": 1021,
+          "humidity": 87,
+          "rain_1h": 2,
+          "clouds": [
+            {
+              "condition": 'NSC'
+            }
+          ]
+        }
+        expect { client.create_measurements([create_params]) }
+          .to raise_error(OpenWeather::Errors::Fault, /expected=string, got=number/)
+      end
+    end
+  end
+
+  describe '#get_measurements' do
+    it 'gets measurements', vcr: { cassette_name: 'stations/get_measurement_success' } do
+      data = client.get_measurements(
+        station_id: '5ed21a12cca8ce0001f1aef1',
+        type: 'd',
+        limit: 100,
+        from: 1469817340,
+        to: 1591620047
+      )
+      expect(data.size).to eq(1)
+      expect(data.first['station_id']).to eq('5ed21a12cca8ce0001f1aef1')
```

Comment:
It seems like the request / response structures for measurements are somewhat different e.g. `wind_speed` vs `wind: { speed: ... }`, I've left it as a raw hash for now. I could make it into a Hashie I suppose?

Reply:
You should make a new structure for anything unique.

Reply:
There would potentially be two structures then, so OpenWeather::Stations::MeasurementRequest / MeasurementResponse? I could try to normalize the differences, but the response isn't as well documented as the request :/

Reply:
I think you should make one, `Measurement` that would match the response since it's more frequently queried I imagine than sent, and convert into the right hash for requests for the OO model.

Reply:
I've made a best effort one :/ I guessed the properties for some of the "submodels" based on personal testing, but there are some weird ones e.g. I created a measurement with `wind_gust` and `wind_speed`, but in the response `wind` is an empty hash. This makes a OO type model a bit tricky.

Reply:
That's fine. You should double-check whether such documentation is really missing on their site, then send them an email asking to document the structures.